### PR TITLE
Emit more metrics from circuit_breaker

### PR DESCRIPTION
**Pull request by onno-vos-dev**
I'd like the capability of tracking times on OK responses.

### CODE REVIEW

**Review by jesperes — APPROVED** [review]

**Comment by onno-vos-dev** [comment]
Apparently the CLA is broken... 
```
You have agreed to the CLA for klarna/circuit_breaker
```

Yet it doesn't show up on cla-assistant.io and I do not have the ability to sign it again... 

Any input?

File: src/circuit_breaker.erl [hunk 1]
```diff
@@ -266,17 +284,21 @@ do_call(Service, CallFun, CallTimeout, ResetFun, ResetTimeout, Thresholds) ->
...
+    event(ok_time_metric, Service, [{time, TimeTaken}])
   catch
     Class:Reason:Stacktrace ->
       exit({raise, Class, Reason, Stacktrace})
   end.
 
 -else.
 
-preserve_exception(CallFun) ->
+preserve_exception(CallFun, Service) ->
```
Comment (mikpe) [inline src/circuit_breaker.erl:301]: `Service` -> `_Service` otherwise pre-OTP21 will get a compile-time warning

Comment (onno-vos-dev) [reply src/circuit_breaker.erl:301]: Actually indicates me totally forgetting support for OTP < 21

**Review by mikpe — APPROVED** [review]

**Comment by onno-vos-dev** [comment]
Tried to create a dummy PR https://github.com/klarna/circuit_breaker/pull/9 to re-trigger the CLA agreement signing and even though I was able to sign it, it still stated that no CLA was signed...

**Comment by mikpe** [comment]
> Tried to create a dummy PR #9 to re-trigger the CLA agreement signing and even though I was able to sign it, it still stated that no CLA was signed...

Do you want to bring that up with our internal open-source people? Otherwise, do you want someone in KCore to merge anyway (as the check is broken and the PR is approved)?

**Comment by onno-vos-dev** [comment]
> > Tried to create a dummy PR #9 to re-trigger the CLA agreement signing and even though I was able to sign it, it still stated that no CLA was signed...
> 
> Do you want to bring that up with our internal open-source people? Otherwise, do you want someone in KCore to merge anyway (as the check is broken and the PR is approved)?

I brought up already and have now pinged a few more people. Let's await a reply...

**Comment by onno-vos-dev** [comment]
Got the OK to just merge it :-) @mikpe can you merge pls?

**Comment by mikpe** [comment]
> Got the OK to just merge it :-) @mikpe can you merge pls?

Merged, tagged as 1.0.3, and imported to our internal mirror.

**Comment by onno-vos-dev** [comment]
Thank you!

**Pull request merged**